### PR TITLE
feat: Switch to string enums for compute

### PR DESCRIPTION
**Pull request by vam-google**
Also update grpc_service configs and introduce gapic_yaml for java clients to override default LRO configuration (which has too way too long timeouts).

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -381,10 +381,12 @@ gapic_generator_ruby_repositories()
 # Discovery
 ##############################################################################
 
+_disco_to_proto3_converter_version = "91953d241824df417b6d4e7c2292e4622b4af432"
```
Comment (alexander-fenster) [inline WORKSPACE:384]: Can we wait for this change to come from the sync?

File: api-index-v1.json [hunk 2]
```diff
@@ -30782,7 +30782,7 @@
       "options": {
         "csharp_namespace": {
           "valueCounts": {
-            "Google.Cloud.Compute.V1Small": 1
+            "Google.Cloud.Compute.V1small": 1
```
Comment (alexander-fenster) [inline api-index-v1.json:30785]: I would expect this one will be autogenerated?

**Review by alexander-fenster — APPROVED** [review]

File: api-index-v1.json (hunk 2 above)
Comment (vam-google) [reply api-index-v1.json:30785]: hm... I don't even know how this got here, I did not touch this file... good catch.###

File: WORKSPACE (hunk 1 above)
Comment (vam-google) [reply WORKSPACE:384]: It does not matter which one comes first, but yes, there is a pending internal cl with this change (you are added as a reviewer)

**Pull request closed without merging**